lib/jekyll/jekyllfeed.rb

```diff
@@ -0,0 +1,50 @@
+require "fileutils"
```

Comment:
This file name feels strange to me, since we also have a `lib/jekyll-feed.rb` as the primary file.

Technically, I believe it should be `lib/jekyll/feed.rb`, with sub-files in `lib/jekyll/feed`, but then also technically, the gem should be required as `jekyll/feed`, however, that's not the design pattern for any of the other gems that I've seen.

Most use `lib/jekyll-feed.rb` as the primary file, with `lib/jekyll-feed/foo.rb` as sub files, which isn't 100% in line with Ruby Gem conventions, but allows us to do `gems: [jekyll-feed]`, and not `gems: [jekyll/feed]`. 

@parkr any strong thoughts one way or the other here?